sdk/browser/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# release 7.9.0 (2022-12-09)
```

Comment:
WDYT to use patch update there, as this changes doesn't introduce any breaking changes (major), no new feature (minor)?

Reply:
ok, i will use patch.